src/logic/operate.js

```diff
@@ -1,9 +1,14 @@
+/* eslint-disable no-console */
```

Comment:
- [ ] <b>REQUIRED:</b> This comment is no longer necessary here and besides, you are not allowed to use `eslint-disable` unless of course when it is totally obvious that there is no other way. kindly remove it.